### PR TITLE
Added kebechet metric and readme

### LINKED ISSUE #393: Exporting metric for total count of repos supported by Kebechet
Export metric for Kebechet

Related PR - 
https://github.com/thoth-station/storages/pull/1773

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Closes #393 
Related #394 

Depended on - https://github.com/thoth-station/storages/pull/1773

### CODE REVIEW

**Comment by goern** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+## Setting up Metrics Exporter locally.
+1. Create a `.env` file from the `.env.template`.
+2. Once you have populated all the values for `.env`, follow up with `pipenv install --dev`.
+3. oc login into the Openshift cluster. 
+5. Run the local version of thoth-storages after restoring the dump - [guide](https://github.com/thoth-station/storages#running-postgresql-locally) (or connect to the production db using replacing the env variables in `.env`).
+4. Run the metrics exporter using - `pipenv run python3 wsgi.py`
+5. You should see metrics in - [localhost:8080](http://localhost:8080)
+
+## Adding new metrics to be exported. 
+1. Add the metric you want to expose to [metrics.py](https://github.com/thoth-station/metrics-exporter/blob/master/thoth/metrics_exporter/metrics.py).
```
Comment (pacospace) [inline README.md:14]: Maybe mention that is based on Prometheus and therefore the metric type need to be from Prometheus available classes from Prometheus package installed (Gauge, Counter, Histogram) which has specific way to be set and specific value need to be provided.

Comment (saisankargochhayat) [reply README.md]: Thanks, I have added the link to Prometheus documentation :smile:

File: thoth/metrics_exporter/jobs/kebechet.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+_LOGGER = logging.getLogger(__name__)
+
+
+class KebechetMetrics(MetricsBase):
+    """Class to evaluate Metrics for Kebechet."""
+
+    @classmethod
+    @register_metric_job
+    def get_active_kebechet_repo_count(cls) -> None:
+        """Get number of repo's Kebechet currently supports."""
```
Comment (pacospace) [inline thoth/metrics_exporter/jobs/kebechet.py:37]: isn't it `repos'`? :)

Comment (saisankargochhayat) [reply thoth/metrics_exporter/jobs/kebechet.py]: changed :stuck_out_tongue:

**Pull request merged**